### PR TITLE
Add a note about tabs at the readme

### PR DESCRIPTION
**Pull request by mrded**
I thought it may be useful to mention the opinion regarding tabs, as it's a very popular question.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,15 @@ If you have doubts about Alacritty's performance or usability, the best way to
...
 
+**_How to do tabs with Alacritty?_**
+
+Alacritty doesn't support tabs by design.
+
+Tabs can be added by Window Managers or Terminal Multiplexers such as tmux.
+
+MacOS users can add tabs using `Prefer tabs when opening documents: Always` setting at `System Settings -> Desctop & Dock`. New windows will be grouped as tabs.
+More info [here](https://support.apple.com/en-gb/guide/mac-help/mchla4695cce/mac).
+
```
Comment (kchibisov) [inline README.md:113]: Recent master mentions tabs in the man page and doesn't require anything special. It also has bindings like `CreateNewTab` just for macOS.

**Review by kchibisov — CHANGES_REQUESTED** [review]

**Comment by kchibisov** [comment]
Not interested in expanding on something like that, we already mention that when refer to multiplexer.

**Comment by mrded** [comment]
Fair enough. It took me two days drilling through closed issues and comments to understand how can I do tabs without using multiplexers on macOS. 

I think a simple note at FAQ may help people like me not asking the same question again and again.

**Comment by mrded** [comment]
Is it documented anywhere?

**Comment by kchibisov** [comment]
In man pages, like I said before. https://raw.githubusercontent.com/alacritty/alacritty/master/extra/man/alacritty-bindings.5.scd

**Comment by mrded** [comment]
Amazing, thank you! Great job guys ❤️ ! 

I'd still recommend mentioning it in the FAQs. It's one of the very often asked questions.

**Pull request closed without merging**